docs/pages/guides/editing-richtext.mdx

```diff
@@ -0,0 +1,118 @@
+
+Let’s discuss if a feature-complete, general purpose rich text input is possible.
+
+React Native allows nested `<Text>` tags, and it even allows them to be used as children of `<TextInput>` to render and edit styled text. It even is synchronous on new react native architecture (the onChange event fires immediately after a new character is typed into the text input field).
+
+Unfortunately, the `<TextInput>` component is built towards working with regular text, and it only returns a string in it’s `onTextChange` callback. This is a significant limitation.
+
+Here’s a quick example. If you are rendering a text input with the following content:
+
+**aa**aa
+
+```jsx
+<TextInput>
+  <Text>
+    <Text style={{ fontWeight: 900 }}>aa</Text>aa
+  </Text>
+</TextInput>
+```
+
+And you append a fifth letter a, the position of your cursor should determine if the new letter is a part of the bold string or not. Unfortunately, the callback only returns `aaaaa`.
+
+There is an additional `onSelectionChange` prop that could be use to get that information, but it makes the task significantly harder. Inserting additional characters (such as newlines for lists or bullet points) also desynchronises the selection.
+
+There are some attempts to build such an editor, such as [markdown-editor](https://github.com/shakogegia/markdown-editor) but no widely used packages exist.
+
+## Markdown editors with always visible styling markers
+
+If you’re ok with using markdown to style text, you can render markdown in a separate, non-editable view while editing. Here’s a package that does that (https://github.com/kunall17/react-native-markdown-editor), but it’s also easy to build on your own using any markdown renderer.
+
+This is an editing experience more suited to power users or programming/tech applications.
+
+You can also explore rendering rich text while showing markdown only in the selected chunk of text or other hybrid approaches.
+
+## Native editors
+
+You can consider using a native Android/iOS rich text editor wrapped into a react-native module. There are a couple of options:
+
+- [react-native-aztec](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-aztec)
+- [gutenberg-mobile](https://github.com/wordpress-mobile/gutenberg-mobile)
+
+You can also wrap any native rich text editor using [expo-modules](/modules/overview/), but if you use different ones on each platform you will need to unify their APIs and input formats somehow.
+
+> Rich text is usually represented using [abstract syntax trees](https://en.wikipedia.org/wiki/Abstract_syntax_tree).
+>
+> For example, a bullet list can be a node of type `bulleted-list` with several children of type `list-item`. You can convert both HTML and markdown to a suitable AST format.
+
+There is also an effort to port the lexical editor to iOS and Android and provide a react-native wrapper. You can track it here: https://github.com/facebook/lexical/discussions/2410.
```

Comment:
```suggestion
There is also an effort to [port the lexical editor to Android and iOS and provide a React Native wrapper](track it here: https://github.com/facebook/lexical/discussions/2410)
```

We can use the link to support the text instead of providing the link directly.